.github/workflows/promote.yml

```diff
@@ -61,6 +66,7 @@ jobs:
         env:
           VERSION: ${{ github.event.inputs.version }}
           CHANNEL: ${{ github.event.inputs.channel }}
+          PRE_RELEASE: ${{ github.event.inputs.pre-release }}
```

Comment:
I hope it works as expected with the input type as `boolean` 🤞

Reply:
I'll find a way to this somehow in my fork...

Reply:
GHA Boolean handling is always tricky, i have to test it out every time too.

Reply:
Tested with [this commit](https://github.com/turkenh/crossplane/commit/ac4728086ebd4eb9fdefa6af8e30a43900f9d77b) and it is working as expected. 

<img width="334" alt="Screenshot 2023-10-26 at 13 41 06" src="https://github.com/crossplane/crossplane/assets/9900707/7e548c54-7508-4dba-9c14-065f0691f3d0">

When the pre-release box is NOT checked:

<img width="428" alt="Screenshot 2023-10-26 at 13 40 31" src="https://github.com/crossplane/crossplane/assets/9900707/8238cf07-3f84-4b74-9617-4e982ad3d811">

When the pre-release box is checked: 

<img width="387" alt="Screenshot 2023-10-26 at 13 40 45" src="https://github.com/crossplane/crossplane/assets/9900707/d7bdc6ce-359b-4013-a652-753e2315a418">